bin/migrate_to_single_table.py

```diff
@@ -132,7 +132,15 @@ def index_record_to_new_table(
                 if not records:
                     break
 
-                records_to_insert = self.get_info_from_mult_tables(records)
+                try:
+                    records_to_insert = self.get_info_from_mult_tables(records)
+                except Exception as e:
+                    raise Exception(
+                        f"""
+                                    Could not insert records with {e} at offset {offset} with the last seen guid {last_seen_guid}. Please re-run the job with the following command
+                                    gen3 job run indexd-single-table-migration-job --start-did {last_seen_guid}
```

Comment:
The service isn't fully aware of the job, so I'd leave the specific details out. I'd just say "Please re-run the job with the following `start-did`: {last_seen_guid}"

Reply:
shouldn't this error handling also be around the bulk_insert_records?